### PR TITLE
Aha/metrics api extension

### PR DESCRIPTION
**Pull request by andyHa**

### CODE REVIEW

File: src/main/java/sirius/biz/analytics/reports/README.md [hunk 1]
```diff
@@ -4,4 +4,10 @@ Provides some helper classes for creating and displaying a table / report based
 in-memory data.
 
 Use the [Report-Tag](../../../../../resources/default/taglib/w/report.html.pasta) 
-to display a [Report](Report.java).
+to display a [Report](Report.java). Note that [SimpleReportBatchJobFactory](../../jobs/interactive/ReportJobFactory.java)
+can be used to provide a simple report which can be started by a user. Note that also
+batch jobs (which cannot be shown interactively as they tke too long to compute) can utilize
```
Comment (sabieber) [inline src/main/java/sirius/biz/analytics/reports/README.md]:
```suggestion
batch jobs (which cannot be shown interactively as they take too long to compute) can utilize
```

File: src/main/java/sirius/biz/analytics/reports/README.md [hunk 2]
```diff
@@ -4,4 +4,10 @@ Provides some helper classes for creating and displaying a table / report based
...
+most of the capabilities by providing a [TableOutput](../../process/output/TableOutput.java)
+which also uses **Cells**. A simple way of creating such a batch job is to extend
```
Comment (sabieber) [inline src/main/java/sirius/biz/analytics/reports/README.md]: Maybe link the Cell class?

File: src/main/java/sirius/biz/analytics/metrics/Metrics.java [hunk 3]
```diff
@@ -150,6 +198,15 @@
      */
     void updateGlobalDailyMetric(String name, int year, int month, int day, int value);
 
+    /**
+     * Creates or updates a daily monthly metric.
```
Comment (meg1502) [inline src/main/java/sirius/biz/analytics/metrics/Metrics.java]:
```suggestion
     * Creates or updates a daily metric.
```

File: src/main/java/sirius/biz/analytics/metrics/Metrics.java [hunk 4]
```diff
@@ -139,6 +177,16 @@
...
+     *
+     * @param target the object for which the metric is stored
+     * @param name   the name of the metric
+     * @param date   the date to determine the year, month and day of the metric
+     * @param value  the value to store
+     */
+    void updateDailyMetric(BaseEntity<?> target, String name, LocalDate date, int value);
+
     /**
      * Creates or updates a daily monthly metric.
```
Comment (meg1502) [inline src/main/java/sirius/biz/analytics/metrics/Metrics.java]:
```suggestion
     * Creates or updates a daily metric.
```

**Pull request merged**